problem-08.c

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
+// Takes a number as input and adds its digits
+
+int main()
+{
+    int number, sum = 0, lastDigit;
+
+    printf("Enter a number: ");
+    scanf("%d", &number);
+
+    for (number; number != 0; number /= 10)

```

Comment:
On my side I am getting this `warning: expression result unused [-Wunused-value]` and it can be resolved as `for (; number != 0; number /= 10)`